setup.py

```diff
@@ -21,6 +21,7 @@
     "newlinejson",
     "apache-airflow==1.9.0",
     "pandas-gbq",
+    "vessel-scoring==1.1"
```

Comment:
@bitsofbits why are we importing `vessel-scoring` in pipe-tools? We should try to avoid circular dependencies.

Reply:
@enriquetuya , I'm not sure how that ended up in there -- bad paste with too many setup.py's open at once I suspect.